example/linux/systemd/talawa-api.service

```diff
@@ -0,0 +1,74 @@
+[Unit]
+# Description of the service
+Description=Talawa-API Service
+
+# Ensure the service starts after the network is available
+After=network.target
+
+[Service]
+# The type of service. 'simple' means the service will start immediately.
+Type=simple
+
+# The command to start the service. This points to the Talawa-api.sh script.Here you can't use TALAWA_API_HOME from shell (~/.bashrc) as systemd will not load any variable. That's why we have to manually add it.Always use absolute path.
+ExecStart=/path/to/your/talawa-api/example/linux/systemd/Talawa-api.sh
+
+# The working directory for the service.User have to export it.
+WorkingDirectory=/path/to/your/talawa-api
```

Comment:
using absoulte path is will best in this case enviroment file is giving error as it will not read properly . so it is good to use direct path . Also it will easy to setup